src/app/(setup)/setup/page.tsx

```
@@ -170,7 +170,7 @@ export default function SetupPage() {
         )
     }
 
-    if (showBrowserNotSupportedModal || showDeviceNotSupportedModal) {
+    if (true) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)